test/cli/test/shortcut.test.ts

```diff
@@ -0,0 +1,49 @@
+import type { TestProject } from 'vitest/node'
+import { resolve } from 'pathe'
+import { describe, expect, it, vi } from 'vitest'
+import { runVitest } from '../../test-utils'
+
+describe('restart browser servers with watch', () => {
+  it('should restart multiple browser servers', async () => {
+    const root = resolve(import.meta.dirname, '../fixtures/browser-multiple')
+    const projects: TestProject[] = []
+    const { vitest, ctx } = await runVitest({ root, watch: true, browser: {
+      enabled: true,
+      provider: 'playwright',
+    }, reporters: [{
+      onBrowserInit(project) {
+        projects.push(project)
```

Comment:
it's a good idea to check this, since this hooks _should_ be called several times - this is what you need to test